scarb-metadata/src/command/internal_command.rs

```diff
@@ -117,12 +116,6 @@ impl InternalScarbCommandBuilder {
         self
     }
 
-    /// Set output format to JSON.
-    pub fn json(&mut self) -> &mut Self {
-        self.json = true;
-        self
-    }
-
```

Comment:
Can we please add `json` flag to `MetadataCommand` and keep it here? This would allow more flexibility in the interface (which would be able to output either json or textual representation.

Reply:
After conversation with @mkaput we decided to remove it, but yes it can be kept.

Reply:
Since you have already added support for it in the parser, I think we should keep it. CC @mkaput